File: javascript/2543.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let [a, b] = lines.shift().trim().split(" ");
let cont = 0;

while(lines.length != 0){

    for(let i = 0; i < a; i++){
        const [id, boolean] = lines.shift().trim().split(" ");
        if(id == parseInt(b) && boolean == 0){
            cont++;
     }
    }
    console.log(cont);

    if(lines.length == 1){
        break;
    }

    [a, b] = lines.shift().trim().split(" ");
    cont = 0;
}

File: java/2543.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.PrintWriter;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        PrintWriter out = new PrintWriter(System.out);
        
        String line;
        
        line = br.readLine();
        if (line == null) return;
        
        String[] firstLine = line.trim().split(" ");
        int a = Integer.parseInt(firstLine[0]);
        int b = Integer.parseInt(firstLine[1]);
        
        int cont = 0;
        
        while (true) {
            
            for (int i = 0; i < a; i++) {
                String[] playerInfo = br.readLine().trim().split(" ");
                int id = Integer.parseInt(playerInfo[0]);
                int presente = Integer.parseInt(playerInfo[1]);
                
                if (id == b && presente == 0) {
                    cont++;
                }
            }
            
            out.println(cont);
            
           
            line = br.readLine();
            if (line == null || line.trim().isEmpty()) {
                break;
            }
            
            String[] nextLine = line.trim().split(" ");
            a = Integer.parseInt(nextLine[0]);
            b = Integer.parseInt(nextLine[1]);
            cont = 0;
        }
        
        out.close();
        br.close();
    }
}
